src/compiler/emitter.ts

```diff
@@ -1921,7 +1921,7 @@ module ts {
                         break;
                     }
                     // _a .. _h, _j ... _z, _0, _1, ...
-                    name = "_" + (tempCount < 25 ? String.fromCharCode(tempCount + (tempCount < 8 ? 0: 1) + CharacterCodes.a) : tempCount - 25);
+                    name = "_" + (tempCount < 25 ? String.fromCharCode(tempCount + (tempCount < 8 ? 0 : 1) + CharacterCodes.a) : tempCount - 25);

```

Comment:
This is pretty confusing. Can you put a comment explaining what this is doing? Also, I think it makes sense to replace the numbers with named constants. This can be addressed separately from this change.